### PR TITLE
unrelated build and other changes/comments needed

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/support_troubleshooting/must_gather_mce.adoc [hunk 1]
```diff
@@ -62,11 +62,13 @@ Complete the following steps to run the `must-gather` command in a disconnected
 
 ----
 REGISTRY=registry.example.com:5000
-IMAGE=$REGISTRY/multicluster-engine/must-gather-rhel8@sha256:ff9f37eb400dc1f7d07a9b6f2da9064992934b69847d17f59e385783c071b9d8
+IMAGE=$REGISTRY/multicluster-engine/must-gather-rhel8@:ff9f37eb400dc1f7d07a9b6f2da9064992934b69847d17f59e385783c071b9d8
```
Comment (swopebe) [inline clusters/support_troubleshooting/must_gather_mce.adoc:66]: @jlpadilla this section (link 64)

File: install/upgrade_disconnected.adoc [hunk 2]
```diff
@@ -9,6 +9,7 @@ See the steps and information to upgrade {product-title} in a disconnected netwo
 == Upgrade from release 2.5 or later
```
Comment (swopebe) [inline install/upgrade_disconnected.adoc:10]: Checking with Install team, line 12.

File: clusters/support_troubleshooting/must_gather_mce.adoc (hunk 1 above)
Comment (jlpadilla) [reply clusters/support_troubleshooting/must_gather_mce.adoc:66]: This is the image SHA, which is like a fingerprint that identifies the image. This information is public when we publish the images.
Maybe it isn't clear that this is only a sample and the user needs to replace it with the correct image that matches the release.
Thanks for checking!

Comment (swopebe) [reply clusters/support_troubleshooting/must_gather_mce.adoc:66]: changes to the line above it reflect that now.

File: install/upgrade_disconnected.adoc (hunk 2 above)
Comment (swopebe) [reply install/upgrade_disconnected.adoc:10]: Slack discussion.

File: clusters/support_troubleshooting/must_gather_mce.adoc [hunk 3]
```diff
@@ -39,10 +39,10 @@ See the following procedure to start using the `must-gather` command:
 
 Check for managed clusters that are not set `True` for the JOINED and AVAILABLE column. You can run the `must-gather` command on those clusters that are not connected with `True` status.
 
-. Add the multicluster engine for Kubernetes image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output:
+. Add the multicluster engine for Kubernetes image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output, with the current supported version replacing `v2x`:
```
Comment (cadawson) [inline clusters/support_troubleshooting/must_gather_mce.adoc]:
```suggestion
. Add the multicluster engine for Kubernetes image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output, with the current supported version replacing `v2.x`:
```

File: install/upgrade_disconnected.adoc [hunk 4]
```diff
@@ -6,9 +6,9 @@ See the steps and information to upgrade {product-title} in a disconnected netwo
 *Note:* This information follows the upgrading procedure in xref:../install/upgrade_hub.adoc#upgrading[Upgrading]. Review that procedure, then see the following information:
 
 [#disconnect-upgrading]
-== Upgrade from release 2.5 or later
+== Upgrade from previous release
 
-During your installation, or upgrade, of {product-title} to release 2.5 or later, you encountered important information that is related to the interdependency between the {product-title} and multicluster engine for Kubernetes operators. See xref:../install/install_disconnected.adoc#install-on-disconnected-networks[Install in disconnected network environments]. Similar considerations are required when you upgrade.
+During your installation, or upgrade, you might encounter important information that is related to the interdependency between the {product-title} and multicluster engine for Kubernetes operators. See xref:../install/install_disconnected.adoc#install-on-disconnected-networks[Install in disconnected network environments] for consideration during install or upgrade.
```
Comment (cadawson) [inline install/upgrade_disconnected.adoc:11]:
```suggestion
During your installation, or upgrade, you might encounter important information that is related to the interdependency between the {product-title} and multicluster engine for Kubernetes operators. See xref:../install/install_disconnected.adoc#install-on-disconnected-networks[Install in disconnected network environments] for considerations during install or upgrade.
```

**Review by cadawson — APPROVED** [review]
A couple of minor suggestions, but looks good. Thanks!

File: install/upgrade_disconnected.adoc (hunk 4 above)
Comment (swopebe) [reply install/upgrade_disconnected.adoc:11]: @cadawson I merged the other, not sure I agree with this.

The subject is the topic (singular) and the predicate needs to agree.

Why would we change this to "consideration(s)"?

Comment (swopebe) [reply install/upgrade_disconnected.adoc:11]: I got rid of this and made it concise: Similar considerations are required when you upgrade. 

Now the use of that word is incorrect, no?

Comment (swopebe) [reply install/upgrade_disconnected.adoc:11]: agreed on slack to keep as a verb, not noun.

**Pull request merged**